packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -112,26 +117,30 @@ class UnthemedCodeEditor extends React.PureComponent<Props> {
 
       readOnly,
       lineNumbersMinChars: 4,
-      lineDecorationsWidth: 0,
+      lineDecorationsWidth: 1 * theme.spacing.gridSize,
```

Comment:
I wanted to use the spacing function, but it outputs a `px` string, whereas this config must be a number.

Reply:
yea, that is fine, for cases like this, it's what this prop is for